Challenge IV/indexIV.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+		<link rel="stylesheet" href="styleIV.css">
+	</head>
+	<body>
+		<p>Name
+  			<input type="text" id="name"><br>
+  			Favorite Food
+  			<input type="text" id="favoriteFood"><br>
+  			Favorite Color
+  			<input type="text" id="favoriteColor"><br>
+  			<button id="button">Submit</button>
+		</p>
+		<script type="text/javascript" src="scriptIV.js"></script>
+		<style type="text/css">
+		td, th {
+    		border: 1px solid #dddddd;
+    		text-align: left;
+    		padding: 8px;
+		}
+		</style>
```

Comment:
Your style code in lines 17-23 should be in a separate css file. You already defined one called "styleIV.css" in line 5 of your code. So, take out the lines here, and include the following ones in your styleIV.css file:

``` css
td, th {
   border: 1px solid #dddddd;
   text-align: left;
   padding: 8px;
}
```